src/Parser.php

```diff
@@ -30,7 +30,7 @@ protected function parseLine(string $line): void
         $element = $this->getElementForLine($line);
     }
 
-    protected function parseInlineElements(string $line, Element &$element): void
+    protected function parseInlineElements(string $line, Element $element): void
```

Comment:
```suggestion
    protected function parseInlineElements(string $line, Element &$element): void
```
Why'd you drop the pass by reference requirement?

Reply:
Objects are always passed as reference.

Reply:
Yes but I want to make sure someone doesn't drop `new Element` in the arg slot.